<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="buttonAddFile">
          <property name="toolTip">
           <string>Add File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="mainwindow.qrc">
            <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelFile">
          <property name="toolTip">
           <string>Delete File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="mainwindow.qrc">
            <normaloff>:/res/delete.png</normaloff>:/res/delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMoveFileUp">
          <property name="toolTip">
           <string>Move File Up</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="mainwindow.qrc">
            <normaloff>:/res/arrow_up.png</normaloff>:/res/arrow_up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMoveFIleDown">
          <property name="toolTip">
           <string>Move File Down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="mainwindow.qrc">
            <normaloff>:/res/arrow_down.png</normaloff>:/res/arrow_down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelBlockUsage">
        <property name="text">
         <string>No memory pack loaded</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="infoWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QComboBox" name="comboProgramType"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Game title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Day</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Month</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBoxMonth">
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="spinBoxDay">
           <property name="maximum">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Program type </string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Ver.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Plays remaining</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QSpinBox" name="spinBoxVersion">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4" colspan="4">
          <widget class="QComboBox" name="comboPlaysLeft"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkFastROM">
         <property name="text">
          <string>Use FastROM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMuteRadio">
         <property name="text">
          <string>Mute SoundLink radio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkRunPSRAM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run from PSRAM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSkipIntro">
         <property name="text">
          <string>Skip St.GIGA intro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkDeleted">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Mark as deleted</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonApply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonReset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelWarning">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionExportAll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/folder.png</normaloff>:/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/disk.png</normaloff>:/res/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>E&amp;xport...</string>
   </property>
  </action>
  <action name="actionExportAll">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/disk_multiple.png</normaloff>:/res/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>Ex&amp;port All...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/page.png</normaloff>:/res/page.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editTitle</tabstop>
  <tabstop>spinBoxMonth</tabstop>
  <tabstop>spinBoxDay</tabstop>
  <tabstop>spinBoxVersion</tabstop>
  <tabstop>comboProgramType</tabstop>
  <tabstop>comboPlaysLeft</tabstop>
  <tabstop>checkFastROM</tabstop>
  <tabstop>checkMuteRadio</tabstop>
  <tabstop>checkRunPSRAM</tabstop>
  <tabstop>checkSkipIntro</tabstop>
  <tabstop>checkDeleted</tabstop>
 </tabstops>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
